### DISCUSSION TITLE
I want to know how to track existing Omniverse Isaac Gym data

### DISCUSSION THREAD

**Discussion by soolmuk** [Toni-SM/skrl#146] [Q&A] [answered]
Hi! I was recently using the Omniverse Isaac Gym and came across the
SKRL library and I'm trying to utilize it.

However, I would like to add the data that are being recorded in the script within Omniverse Isaac Gym using the

SKRL's track_data, but I'm not sure how to do it.

For example, if I use pytorch_crazyflie_ppo.py, which is provided by default with SKRL, it writes to tensorboard like below.
![image](https://github.com/Toni-SM/skrl/assets/72430756/6c716ed4-07d2-44e0-b3ba-377928cc2540)

However, if you run the Crazyflie example provided by the Omniverse Isaac Gym, you'll see the following.
![image](https://github.com/Toni-SM/skrl/assets/72430756/32edd81f-4711-40f8-910b-82836e1c4117)

Is there any way I can track datas like episode/rew_spin and pos?

I would really appreciate it if you could let me know.

**Comment by Toni-SM** [comment, marked as answer]
Hi @soolmuk 

I started, in the [`log_episode_info`](https://github.com/Toni-SM/skrl/tree/log_episode_info) branch, the implementation to log environment info. The current implementation is done for the PyTorch backend

![image](https://github.com/Toni-SM/skrl/assets/22400377/cbce9fed-c7c2-4560-8909-3524a027781b)

**Reply by Toni-SM** [reply]
After adding the implementation for the JAX backend, I will merge it to the `develop` branch

**Reply by soolmuk** [reply]
How nice! Thanks! I'll test my work with this branch today!!
